packages/wrangler/src/__tests__/index.test.ts

```diff
@@ -160,11 +178,46 @@ describe("wrangler", () => {
       expect(packageJson.version).toEqual("1.0.0");
     });
 
-    it("should not touch an existing package.json in an ancestor directory", async () => {
-      mockConfirm({
-        text: "Would you like to use typescript?",
-        result: false,
+    it("should offer to install wrangler into an existing package.json", async () => {
+      mockConfirm(
+        {
+          text: "Would you like to install wrangler into your package.json?",
+          result: true,
+        },
+        {
+          text: "Would you like to use typescript?",
+          result: false,
+        }
+      );
+
+      fs.writeFileSync(
+        "./package.json",
+        JSON.stringify({ name: "test", version: "1.0.0" }),
+        "utf-8"
+      );
+
+      await runWrangler("init");
+      const packageJson = JSON.parse(
+        fs.readFileSync("./package.json", "utf-8")
+      );
+      expect(packageJson.name).toEqual("test");
+      expect(packageJson.version).toEqual("1.0.0");
+      expect(packageJson.devDependencies).toEqual({
+        wrangler: expect.any(String),
       });
+    });
+
+    it("should not touch an existing package.json in an ancestor directory", async () => {
+      mockConfirm(
+        {
+          text: "Would you like to install wrangler into your package.json?",
+          result: false,
+        },
+        {
+          text: "Would you like to use typescript?",
+          result: false,
+        }
+      );
```

Comment:
Should this test also check that the ancestor package.json does not contain `wrangler` in the devDeps?

Reply:
Probably not. The codepath for checking the innards of a resolved package.json isn't going to change, so another test here wouldn't test anything different. 

The thing to do, I think, is to modify `mockConfirm` and make sure all the expectations are called, once, in order, in a given test.

Reply:
What I meant is that (following the description of this test) we should check that the ancestor `package.json` does not get modified by us.

Reply:
aight, I can do that